app/controllers/completed_applications_controller.rb

```diff
@@ -29,12 +29,18 @@ def amend
 
   def pagination_params
     {
-      sort: params[:sort],
+      sort: sort_direction,
       page: params[:page],
       per_page: Kaminari.config.default_per_page,
     }
   end
 
+  def sort_direction
+    return nil if params[:sort].blank?
+
+    "#{params[:sort]}ending"
```

Comment:
This feels a bit sketchy 😅 

Maybe we can keep a map of what internally we use (`asc` / `desc`) and what the datastore uses (`ascending` / `descending`). It is sad we couldn't keep this consistent in the datastore. And perhaps hide that logic or mapping in the service object, `GetApplications`.